### PR TITLE
Exploring AUL integration within unit tests

### PR DESCRIPTION
**Pull request by akornich**
## Description of the change

> Exploring AUL integration within unit tests

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> n/a

### Development

- [x] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [x] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Review by pawelsz-rb — COMMENTED** [review]
looks good, just wondering if we could be consistent and map RollbarLevel_Warning to OSLogEntryLogLevelWarning and RollbarLevel_Critical to OSLogEntryLogLevelCritical

Also, what is the benefit of leaving commented out legacy working version in  RollbarAulTests.m  ?

File: RollbarNotifier/Tests/RollbarNotifierTests-ObjC/RollbarAulTests.m [hunk 1]
```diff
@@ -0,0 +1,414 @@
...
+        return;
+    }
+    
+    NSDateFormatter *formatter = [[NSDateFormatter alloc] init];
+    [formatter setDateFormat:@"YYYY-MM-dd 'at' HH:mm:ss.SSSSSSXX"];
+
+    snapshot[@"date"] = [formatter stringFromDate:entry.date];
+    snapshot[@"className"] = entry.className;
+    snapshot[@"composedMessage"] = entry.composedMessage;
+    //snapShot[@""] = entry.storeCategory;
```
Comment (pawelsz-rb) [inline RollbarNotifier/Tests/RollbarNotifierTests-ObjC/RollbarAulTests.m:146]: this can probably be removed

Comment (akornich) [reply RollbarNotifier/Tests/RollbarNotifierTests-ObjC/RollbarAulTests.m:146]: That is, for now, to keep as a reminder that there is an extra attribute that we may or may not want to consider capturing. In general, this PR is an experimental playground that will be replaced with unit test implementation that would actually be testing production level AUL integration once it is productized.

**Comment by akornich** [comment]
> looks good, just wondering if we could be consistent and map RollbarLevel_Warning to OSLogEntryLogLevelWarning and RollbarLevel_Critical to OSLogEntryLogLevelCritical

OSLogEntryLogLevel enum does not define OSLogEntryLogLevelWarning. The closest to our RollbarLevel_Warning  would be the OSLogEntryLogLevelNotice value.

> 
> Also, what is the benefit of leaving commented out legacy working version in RollbarAulTests.m ?

I would prefer to keep this initial simplified implementation for reference until we have a product-level implementation of the AUL integration.

**Review by pawelsz-rb — APPROVED** [review]
I talked to @akornich offline, for now it's ok to approve

**Comment by akornich** [comment]
@pawelsz-rb , thank you!

**Pull request merged**